projects/demo/src/modules/experimental/badge/badge.template.html

```diff
@@ -0,0 +1,164 @@
+<tui-doc-page
+    header="Badge"
+    package="EXPERIMENTAL"
+    type="components"
+>
+    <ng-template pageTab>
+        <tui-notification status="warning">
+            This code is
+            <strong>experimental</strong>
+            and is a subject to change. Expect final solution to be shipped in the next major version
+        </tui-notification>
+
+        <p>New version for Badge component following updated design specs for displaying text, pictures and icons.</p>
+
+        <tui-doc-example
+            id="base"
+            heading="Basic"
+            [content]="example1"
+        >
+            <tui-badge-example-1></tui-badge-example-1>
+        </tui-doc-example>
+
+        <tui-doc-example
+            id="sizes"
+            heading="Sizes"
+            [content]="example2"
+        >
+            <tui-badge-example-2></tui-badge-example-2>
+        </tui-doc-example>
+
+        <tui-doc-example
+            id="variants"
+            heading="Variants (mobile platform)"
```

Comment:
```suggestion
            heading="Content type (mobile platform)"
```

___
The same naming as in API page
<img width="790" alt="Screenshot 2023-09-01 at 11 20 02" src="https://github.com/taiga-family/taiga-ui/assets/35179038/6a97861a-6119-4fc8-a044-f737ea819606">